specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/datafactory.json

```diff
@@ -375,6 +375,93 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/sharedFactory": {
+      "post": {
+        "tags": [
+          "factories"
+        ],
+        "operationId": "Factories_ListSharedFactory",
+        "x-ms-examples": {
+          "Factories_ListSharedFactory": {
+            "$ref": "./examples/Factories_ListSharedFactory.json"
+          }
+        },
+        "description": "List data factories which the given data factory has access.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/factoryName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/FactoryListResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Data Factory service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/sharedIntegrationRuntime": {
+      "post": {
+        "tags": [
+          "factories"
+        ],
+        "operationId": "Factories_ListSharedIntegrationRuntime",

```

Comment:
This operation should use the x-ms-pageable extension

Reply:
@annatisch Done.